<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="hardware_trigger">
         <property name="toolTip">
          <string>Configure to hardware trigger</string>
         </property>
         <property name="text">
          <string>Hardware Trigger</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="software_trigger">
         <property name="toolTip">
          <string>Configure to software trigger</string>
         </property>
         <property name="text">
          <string>Software Trigger</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="freerun">
         <property name="text">
          <string>Freerun</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="acquisition">
         <property name="toolTip">
          <string>Start</string>
         </property>
         <property name="text">
          <string>Start/Stop</string>
         </property>
         <property name="icon">
          <iconset theme="media-playback-start">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_snap">
         <property name="toolTip">
          <string>Take a single image</string>
         </property>
         <property name="text">
          <string>Snap</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/qtutils/fugue/control-stop.png</normaloff>:/qtutils/fugue/control-stop.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="settings_tool">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../clones/qtutils/qtutils/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
